chainer/training/extensions/polynomial_shift.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import division
+
+from chainer.training import extension
+
+
+class PolynomialShift(extension.Extension):
+
+    """Trainer extension to polynomially shift an optimizer attribute.
+
+    This extension polynomially decreases the specified attribute of the
```

Comment:
"increases or decreases"
(if `rate` is negative, it will be an increasing function.)